### ISSUE TITLE
Unable to edit the alt text after sending

### ISSUE THREAD

**Issue by Indeedornot** [mastodon/mastodon#26309]
### Steps to reproduce the problem

1. Send a post with an image
2. Go into editing the post and change alt text 
3. Try to save
...

Previously this issue didn't appear 

https://github.com/mastodon/mastodon/assets/49447848/46ec70bc-37ce-42ec-9fb8-780aa98b96d7

Video of the mentioned issue

It was also mentioned by members of other instances, since I tend to ask people to edit their posts to add alt text 

### Expected behaviour

The post should be able to save and the alt text should be updated

### Actual behaviour

There appears a "Record not found" error

### Mastodon instance

hachyderm.io

### Mastodon version

V4.1.4

**Comment by tfardet** [comment]
@Indeedornot Somebody on https://scicomm.xyz just reported this same issue on Mastodon for Android v1.1.4 (45), so this might be misplaced (I cannot tell from the video if the OP was also using the app).
I didn't find anything related on the android issue tracker, should I open a new issue there?

**Comment by Indeedornot** [comment]
Haven't used mastodon for some time, but
Android Mastodon App Version 2.1.6
Server hachyderm.io Version 4.2.0

The issue does not appear there anymore

Same cannot be said for Megalodon Android App 1.2.3

It might've been fixed in the official app

(Note the original issue was for Mastodon app, just to avoid any confusion)

**Comment by tfardet** [comment]
OK, so this is in the wrong repo.
Our instance is still on 4.1 so it may be solved in 4.2, would be good to have a bugfix for 4.1 though, many instance have not had time to upgrade yet.
